frontend/e2e/pages/metrics/config-step.ts

```diff
@@ -252,6 +255,22 @@ export class ConfigStep {
     await this.toDateInput.nth(number).fill(endDate);
   }
 
+  async typeInMultipleRanges(ranges: { startDate: string; endDate: string }[]) {
+    for (let i = 0; i < ranges.length; i++) {
+      const dateRange = {
+        startDate: format(ranges[i].startDate),
+        endDate: format(ranges[i].endDate),
+      };
+      if (i) {
+        await this.addNewTimeRange();
+        await expect(this.timeRangeRows).toHaveCount(i + 1);
+      }
+      await this.typeInDateRange({ ...dateRange, number: i });
+      await expect(this.fromDateInput.nth(i)).toHaveValue(dateRange.startDate);
+      await expect(this.toDateInput.nth(i)).toHaveValue(dateRange.endDate);
```

Comment:
Follow the POM of e2e, not add assertion in the action method.

Reply:
done!